src/hub/atlas.js

```diff
@@ -421,6 +421,73 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint PATCH method
+     * 
+     * @returns {Boolean} indicates whether the patch was successful or not
+     */
+    async patchUser(data) {
+        const op = { method: "PATCH", body: { ...data } };
+        try{
+            await this.fetch("/v1/user/" + data.id, op);
+            return true;
+        }catch (err){
+            console.log(err);
+            return false;
+        }
+    }
+
+    /**
+     * Add a new user to the existing user set
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint POST method
+     * 
+     * @returns {Boolean} indicates whether the post was successful or not
+     */
+    async postUser(data) {
+        const op = { method: "POST", body: { ...data } };
+        try{
+            await this.fetch("/v1/user/", op);
+            return true;
+        }catch (err){
+            console.log(err);
+            return false;
+        }
+    }
+
+    /**
+     * Register a new user with atlas
+     *
+     * @param {Object} body - A user data object in the form:
+     *  {   
+     *      captcha, 
+     *      phone, 
+     *      email, 
+     *      fullname, 
+     *      tag_slug, 
+     *      password, 
+     *      org_name, 
+     *      org_slug 
+     * }
+     * All fields are required except phone
+     * 
+     * Use the site key 6Lcr4JMUAAAAAOljN5puqdFrcVeCyexMNHlWtWHX as the seed
```

Comment:
This can't be checked in.  We will may have to rotate keys now actually since this is in the permanent records of github.

Reply:
Thats the public key. Anybody who wants to use that function will need that key.

Reply:
The only thing that key will allow you to do is create a recaptcha as part of a sign up form. The recaptcha will generate a token using that key which is in turn sent to whatever service (atlas in this case) where it is sent to google's recaptcha service along with the secret key which is stored in an environment variable on the server. Google's service then validates the token and sends a response. Anybody who wants to hit the /join/ endpoint will require that site key.

Reply:
I know they'll need A public key, but they need to deal with us directly before we just hand it out.